src/deployment-guides/overview.md

```diff
@@ -1,7 +1,7 @@
 # Deploy Tanzu for Kubernetes Operations
 
-The following provide deployment instructions for the Tanzu for Kubernetes operations reference designs. The deployments use example configuration values. Therefore, make sure to use configuration values that are appropriate for your environment. 
+The following provide deployment instructions for the Tanzu for Kubernetes operations reference designs. The deployments use example configuration values. Therefore, make sure to use configuration values that are appropriate for your environment.
 
 - [Deploy Tanzu for Kubernetes Operations on AWS](tko-aws.md): Outlines the steps for deploying VMware Tanzu for Kubernetes operations on AWS. The deployment is based on the reference design provided in [VMware Tanzu for Kubernetes Operations on AWS Reference Design](../reference-designs/tko-on-aws.md).
+- [Deploy Tanzu for Kubernetes Operations using vSphere with Tanzu](tko-on-vsphere-with-tanzu-deploy.md): Outlines the steps for deploying VMware Tanzu for Kubernetes operations on vSphere with Tanzu in a vSphere environment backed by a Virtual Distributed Switch (VDS) and leveraging NSX Advanced Load Balancer (ALB) for L4/L7 load balancing & ingress.
```

Comment:
I am a bit confused by the naming here, in that "with Tanzu" seems redundant. Is there a distinction that we are trying to make?

Can we make this "Deploy Tanzu for Kubernetes Operations on vSphere" and rename the files (markdown and images) accordingly?

Reply:
While it seems redundant, I believe per the marketing guidelines the way it is written is actually correct. Where as "Deploy Tanzu for Kubernetes Operations on vSphere" would be ambiguous in that it might be TKG Multi-Cloud as well.  @KeithRichardLee might be able to provide some insight here.

Reply:
vSphere with Tanzu is a separate implementation that just vSphere. See https://docs.vmware.com/en/VMware-vSphere/7.0/vmware-vsphere-with-tanzu/GUID-C163490C-BE03-4DFE-8A03-5316D3245765.html.

Reply:
:+1: didn't realize this. All good on the rename.